src/main/resources/validationrules/civil/civil-judicial-org-role-mapping.drl

```diff
@@ -0,0 +1,266 @@
+package validationrules.civil;
+
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.RoleAssignment;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.ActorIdType;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.RoleCategory;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.RoleType;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.Classification;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.GrantType;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.JudicialAccessProfile;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.JudicialOfficeHolder;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.JudicialBooking;
+import java.util.ArrayList;
+import  uk.gov.hmcts.reform.orgrolemapping.util.JacksonUtils;
+import java.util.HashMap
+import java.util.Map
+import com.fasterxml.jackson.databind.JsonNode;
+import uk.gov.hmcts.reform.orgrolemapping.domain.model.enums.FeatureFlagEnum;
+import function uk.gov.hmcts.reform.orgrolemapping.domain.service.RequestMappingService.logMsg;
+
+
+/*
+ * CIVIL "judge" Org role mapping.
+ */
+rule "civil_judge_org_role"
+when
+  $f:  FeatureFlag(status && flagName == FeatureFlagEnum.CIVIL_WA_1_0.getValue())
+  $joh: JudicialOfficeHolder(office in ( "CIVIL District Judge-Salaried", "CIVIL Presiding Judge-Salaried",
+                                         "CIVIL Resident Judge-Salaried", "CIVIL Deputy District Judge-Fee-Paid",
+                                         "CIVIL Deputy District Judge - Sitting in Retirement-Fee-Paid",
+                                         "CIVIL Recorder-Fee-Paid"))
+then
+   Map<String,JsonNode> attribute = new HashMap<>();
+      if($joh.getOffice().contains("Fee-Paid")) {
+         attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Fee-Paid"));
+      } else {
+         attribute.put("contractType", JacksonUtils.convertObjectIntoJsonNode("Salaried"));
+      }
+   attribute.put("jurisdiction", JacksonUtils.convertObjectIntoJsonNode("CIVIL"));
```

Comment:
Not priority, joh.getJurisdiction have CIVIL value which is set on civil-judicial-office-holder-mapping.drl. can also be used here instead of hardcoding.